controllers/tinkerbell_datacenter_controller.go

```diff
@@ -0,0 +1,20 @@
+package controllers
+
+import (
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// TinkerbellDatacenterReconciler reconciles a TinkerbellDatacenterConfig object.
+type TinkerbellDatacenterReconciler struct {
```

Comment:
Does datacenter controller need a validator?

Reply:
we will include the validations once we start the implementation, this is more of a skeleton to build the controller.